WordPress/Classes/ViewRelated/Comments/EditReplyViewController.m

```diff
@@ -11,6 +40,122 @@ - (void)viewDidLoad
+        [self.view addConstraints:[NSLayoutConstraint constraintsWithVisualFormat:@"V:|-30.0-[suggestionsview]"
+                                                                          options:0
+                                                                          metrics:nil
+                                                                            views:views]];
+        
+        // Pin the suggestions view bottom to the bottom of the reply box
+        [self.view addConstraint:[NSLayoutConstraint constraintWithItem:self.suggestionsTableView
+                                                              attribute:NSLayoutAttributeBottom
+                                                              relatedBy:NSLayoutRelationEqual
+                                                                 toItem:self.textView
+                                                              attribute:NSLayoutAttributeBottom
+                                                             multiplier:1
+                                                               constant:0]];
+        
+        // listen to textView changes
+        self.textView.delegate = self;
+    }
+}
+
+#pragma mark - UITextViewDelegate
+
+- (BOOL)textView:(UITextView *)textView shouldChangeTextInRange:(NSRange)range replacementText:(NSString *)text
+{
+    // Get all the text
+    NSString *textViewText = textView.text;
+    NSRange textViewRange = NSMakeRange(0, range.location);
+    NSString *pretext = [[textViewText substringWithRange:textViewRange] stringByAppendingString:text];
+    NSArray *words = [pretext componentsSeparatedByCharactersInSet:[NSCharacterSet whitespaceAndNewlineCharacterSet]];
+    [self.suggestionsTableView showSuggestionsForWord:[words lastObject]];
+    
+    return YES;
+}
+
+#pragma mark - SuggestionsTableViewDelegate
+
+- (void)suggestionsTableView:(SuggestionsTableView *)suggestionsTableView didChangeTableBounds:(CGRect)bounds
+{
+    CGRect textViewBounds = self.textView.bounds;
+    CGRect caretRect = [self.textView caretRectForPosition:self.textView.selectedTextRange.end];
+    
+    // If we are showing suggestions, mind the bottom of the caret, to make sure the typing
+    // insertion point is covered by the suggestions below
+    
+    if (bounds.size.height > 0) {
+        CGPoint contentOffset = [self.textView contentOffset];
+        
+        // If we haven't cached the contentOffset already, do so now
+        if (!self.hasCachedContentOffset) {
+            self.hasCachedContentOffset = true;
+            self.cachedContentOffset = contentOffset;
+        }
+        
+        CGFloat caretBottom = caretRect.origin.y + caretRect.size.height - contentOffset.y;        
+        CGFloat gapBottom = textViewBounds.size.height - bounds.size.height;
+        if (caretBottom > gapBottom) {
+            contentOffset.y += (caretBottom - gapBottom) + 10.; // plus a little more
```

Comment:
(Sorry, i've missed this one last time!) can we move the `10.` to a constant instead?